xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/test/java/org/xwiki/rendering/internal/macro/IncludeMacroTest.java

```diff
@@ -536,4 +536,46 @@ private XDOM getXDOM(String content) throws Exception
 
         return blocks;
     }
+    
+    @Test
+    void executeIncludeMacroWhenExcludeFirstHeadingTrue() throws Exception 
+    {
+        // Expected content containing no heading.
+        // @formatter:off
+        String expected = "beginDocument\n"
+            + "beginMetaData [[source]=[wiki:space.document][syntax]=[XWiki 2.0]]\n"
+            + "beginSection\n"
+            + "beginSection\n"
+            + "beginParagraph\n"
+            + "onWord [content]\n"
+            + "endParagraph\n"
+            + "endSection\n"
+            + "endSection\n"
+            + "endMetaData [[source]=[wiki:space.document][syntax]=[XWiki 2.0]]\n"
+            + "endDocument";
```

Comment:
Why are you expecting this? Why expect some sections? Why expect 2 sections? What are you testing?